src/main/java/org/web3j/protocol/core/personal/JsonRpc2_ExtendedWeb3j.java

```diff
@@ -0,0 +1,119 @@
+package org.web3j.protocol.core.personal;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.web3j.protocol.Web3jService;
+import org.web3j.protocol.core.JsonRpc2_0Web3j;
+import org.web3j.protocol.core.Request;
+import org.web3j.protocol.core.personal.methods.response.EcRecovered;
+import org.web3j.protocol.core.personal.methods.response.ImportedRawKey;
+import org.web3j.protocol.core.personal.methods.response.LockedAccount;
+import org.web3j.protocol.core.personal.methods.response.NewAccount;
+import org.web3j.protocol.core.personal.methods.response.Signed;
+import org.web3j.protocol.core.personal.methods.response.Transaction;
+import org.web3j.protocol.core.personal.methods.response.UnlockedAccount;
+import org.web3j.protocol.parity.methods.response.PersonalListAccounts;
+
+public class JsonRpc2_ExtendedWeb3j extends JsonRpc2_0Web3j
```

Comment:
Once the Interface has been split (see comment below). I recommend you name the implementation class files:

- org.web3j.protocol.admin.JsonRpc2_0Admin
- org.web3j.protocol.geth.JsonRpc2_0Geth